packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -850,6 +975,18 @@ describe("r2", () => {
 		`);
 			});
 
+			it("should upload R2 object with storage class to bucket", async () => {
```

Comment:
Does this test run against real R2? If that's the case, we'll have to remove it as it's currently private preview.

Reply:
Ah, it doesn't look like it -- it's using mock api and token. I think we're good.